### ISSUE TITLE
incoming-pr branch is outdated, so that it is impossible to foollow guidelines for contributing

### ISSUE THREAD

**Issue by evilaliv3** [axemclion/grunt-saucelabs#198]
incoming-pr branch, cited by [guidelines for contributing](https://github.com/axemclion/grunt-saucelabs/blob/master/CONTRIBUTING.md)

i think you should take care of changin the guidelines if no more needed or rebase the incoming-pr branch to latest master.

spotted while trying to following the guidelines for opening the pull request: https://github.com/axemclion/grunt-saucelabs/pull/197

\cc @axemclion @Jonahss

**Comment by axemclion** [comment]
@Jonahss Should we revise the guidelines ? I think we should simply ask for PRs to be against master. I had originally done the `incoming-pr` branch so as to not break master, and have the sauce key private, but clearly that did not work.

**Comment by Jonahss** [comment]
Yeah we can just edit the docs and go to a regular PR to master system.
